esmvaltool/recipes/hydrology/recipe_globwat.yml

```diff
@@ -0,0 +1,98 @@
+# ESMValTool
+# globwat.yml
+---
+documentation:
+  description: |
+    Recipe pre-processes ERA-Interim and ERA-5 data
+    for use in the GlobWat hydrological model.
+    GlobWat is a Global Hydrological Model provided by FAO
+    http://www.fao.org/nr/water/aquamaps/
+
+    By default, PET is calculated by the De Bruin method. However, users
+    can choose the Langbein method by setting the langbein_pet in the recipe
+    to True.
+
+    More information about the De Bruin method can be found at:
+    https://doi.org/10.1175/JHM-D-15-0006.1
+    More information about the Langbein method can be found at:
+    https://doi.org/10.3133/cir52
+
+    The target file has been generated from one of the models sample files
+    (prc01wb.asc). The file can be found at:
```

Comment:
How did you generate this? Do you have a github repo containing the script?

Reply:
You can either download the generated target file from the following repository or use the GDAL translator to convert the file from ASCII format to NetCDF format by entering the following command into the terminal:
gdal translate -of netCDF ASCII prc01wb.asc globwat_target_grid.nc https://github.com/babdollahi/globwat_target_grid 
Link to the repository including the target file and used method added to the recipe.